charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 3.34.0
```

Comment:
This is probably not the right version, it doesn't make sense to merge this until we set the default agent version to `7.47.0` which still works with the old single-hyphen arguments.

Reply:
I do suggest to add to the helm chart conditions based on the agent version

Reply:
You're absolutely right, made it backward compatible.